### DISCUSSION TITLE
Run Child Workflow in Blocking mode

### DISCUSSION THREAD

**Discussion by Rummy99** [elsa-workflows/elsa-core#3049] [Q&A]
I am trying to run a child workflow which is expected to go to suspended state, waiting for http endpoint. The parent which calls child in Blocking mode using Run Workflow activity, couldn't be resumed. 
Is it possible to run Child Workflow in this manner. 

I have provided a sample screenshot of child  workflow. 

![PXL_20220520_060822644.jpg](https://user-images.githubusercontent.com/85660159/169464100-0f36f4e2-0903-4177-9a5f-de666dbdd392.jpg)

**Comment by sfmskywalker** [comment]
Hi @Rummy99 

I think this should work, so I'm not sure why it doesn't.
Perhaps you can open an issue with two sample workflows (parent and child) that reproduces this problem?
